src/app/players/base-player/base-player.component.ts

```diff
@@ -52,10 +52,10 @@ export class BasePlayerComponent implements OnDestroy {
   }
 
   playerEvents(event) {
-    if (event?.edata?.type === 'SHARE') {
+    if (event?.detail?.edata?.type === 'SHARE') {
```

Comment:
Use always global constants for any strings.

Reply:
Done